main.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"code-review-bot/internal"
+	"code-review-bot/internal/web"
+	"code-review-bot/internal/web/routes"
+	"context"
+	"strings"
```

Comment:
Error handling in `initializeGithubClient` does not account for a potential user not being returned, which may lead to a nil pointer dereference when calling `user.GetLogin()`. Ensure to check if `user` is not nil before accessing its methods.